### DISCUSSION TITLE
No bbregister_bold file but boilerplate says bbregister was run

### DISCUSSION THREAD

**Discussion by Jolinda** [nipreps/fmriprep#3275] [Q&A] [answered]
Hi, we have tried running v.23.2.1 (and .0) with a couple of different data sets. None of the outputs include any *desc-bbregister_bold.svg  files; instead we have *desc-coreg_bold.svg. It's my understanding that this indicates that bbregistration failed and the fallback to coreg was used (https://neurostars.org/t/fmriprep-coreg-figures/4334/3). However, the summary and boilerplate still indicate that bbregister was used. Has there been a change to fmriprep, so that it no longer creates a *desc-bbregister_bold.svg file? Or is it silently switching to another registration method without changing the boilerplate text? If it's the latter, I see this as a huge problem: people are going to believe and report that they used a registration method that they did not in fact use.

If we should in fact be getting *desc-bbregister_bold.svg files: has anyone gotten them with the latest version of fmriprep? Does anyone have any ideas why it might not be working for us? We are running on an HPC using apptainer.

**Comment by effigies** [comment, marked as answer]
The naming of SVGs no longer reflects the specific coregistration tool that generated the transformation. That's an implementation detail that was used to debug a particular case.

The report should be correct.

**Comment by Jolinda** [comment]
Thank you! This is a relief.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
